packages/airnode-validator/test/validator.feature.ts

```diff
@@ -0,0 +1,44 @@
+import { execSync, spawnSync } from 'child_process';
+import { join } from 'path';
+
+const runValidator = (args: string[]) => {
+  const command = ['node', join(__dirname, '../dist/bin/validator.js'), ...args].join(' ');
+
+  return spawnSync(command, { shell: true });
+};
+
+describe('validator CLI', () => {
+  beforeAll(() => {
+    execSync('yarn build');
+  });
+
+  it('shows help', () => {
+    const cliHelp = runValidator(['--help']).stdout.toString();
+
+    expect(cliHelp).toMatchSnapshot();
+  });
+
+  it('validates valid configuration', () => {
+    const args = [
+      `--config ${join(__dirname, './fixtures/valid-config.json')}`,
+      `--secrets ${join(__dirname, './fixtures/valid-secrets.env')}`,
+    ];
+
+    const output = runValidator(args).stderr.toString();
+
+    expect(output).toEqual('✔ The configuration is valid\n');
```

Comment:
I would prefer not to have weird unicode characters in the codebase. They are weirdly displayed if your font in the editor doesn't support them and I don't think it brings much value in this case.

Reply:
I think I had problems with these tests so I think I've updated it in the later PR to just `expect.stringContaining`. Will port these changes here as well.

Reply:
Done.